packages/flutter_tools/test/general.shard/web/devices_test.dart

```diff
@@ -379,6 +384,41 @@ void main() {
 
     expect((await macosWebDevices.pollingGetDevices()).whereType<MicrosoftEdgeDevice>(), isEmpty);
   });
+
+  @isTest
```

Comment:
I'm not sure this needs to be marked with this annotation. I think it's sufficient to just invoke `testWithoutContext` inside this function if we want to share the test implementation across scenarios.

Reply:
Yes, `testWebLaunchUrl` needs to be marked with `@isTest`. Otherwise the IDE won't identify the tests below.

For example, when the annotation is present:
<img width="499" height="117" alt="image" src="https://github.com/user-attachments/assets/0efb71d3-45a7-4d33-9b53-f0b323db22ec" />

If the annotation is removed, the status, run and debug buttons don't show anymore:
<img width="465" height="104" alt="image" src="https://github.com/user-attachments/assets/03c9a8b0-5368-46c4-88ee-f9f0fc0f507c" />